### PR TITLE
CCTRI-849/ add judgement mapping

### PR DESCRIPTION
**Pull request by GeorgiyII**

### CODE REVIEW

File: api/enrich.py [hunk 1]
```diff
@@ -101,6 +102,54 @@ def extract_verdicts(outputs, start_time):
     return docs
 
 
+def extract_judgement(outputs):
```
Comment (SugarrayPro) [inline api/enrich.py]: How about Dictionary Comprehension?

File: api/enrich.py [hunk 2]
```diff
@@ -101,6 +102,54 @@ def extract_verdicts(outputs, start_time):
...
+    docs = []
+
+    for output in outputs:
+
+        reports = output['data']['reports']
+        reports.sort(key=lambda x: x['reportedAt'], reverse=True)
+
+        if len(reports) >= 100:
```
Comment (SugarrayPro) [inline api/enrich.py]: Magic variable

File: api/enrich.py [hunk 3]
```diff
@@ -101,6 +102,54 @@ def extract_verdicts(outputs, start_time):
...
+            reports = reports[:100]
+
+        for report in reports:
+
+            start_time = datetime.strptime(
+                report['reportedAt'].split('+')[0],
+                '%Y-%m-%dT%H:%M:%S'
+            )
+            end_time = start_time + timedelta(days=7)
```
Comment (SugarrayPro) [inline api/enrich.py]: Magic variable

File: api/enrich.py (hunk 2 above)
Comment (GeorgiyII) [reply api/enrich.py]: done

File: api/enrich.py (hunk 3 above)
Comment (GeorgiyII) [reply api/enrich.py]: done

File: api/enrich.py (hunk 1 above)
Comment (GeorgiyII) [reply api/enrich.py]: will be poorly readable

**Pull request merged**